rfcs/text/0021-threat-enrichment.md

```diff
@@ -69,6 +198,10 @@ The goal here is to research and understand the impact of these changes on users
 Stage 1: Identify potential concerns, implementation challenges, or complexity. Spend some time on this. Play devil's advocate. Try to identify the sort of non-obvious challenges that tend to surface later. The goal here is to surface risks early, allow everyone the time to work through them, and ultimately document resolution for posterity's sake.
 -->
 
+I believe this format should actually simplify much of the enrichment logic originally proposed in RFC 0008, since a naive implementation would simply copy the fields directly from each indicator document into `threat.enrichments`, and add the appropriate `matched` fields.
+
+While not a concern for ECS consumers at large, our existing experimental implementation within Kibana Security Solution will have to change significantly (for the better!), with accompanying data migration: https://github.com/elastic/security-team/issues/1102
```

Comment:
Is there an accompanying public issue or a summary that we can include here instead of linking to the private issue?

Reply:
I will create a public issue for this.

Reply:
https://github.com/elastic/kibana/issues/100510